File: python/1_read.py
# -*- coding: utf-8 -*-
"""
Filename: 1_read.py


Purpose:	This .py file reads auto.dta


Created by: adw54
Created on: 7 Oct 2020
Last updated on: 7 Oct 2020
"""

import os
import pandas as pd
os.chdir(root)

df = pd.read_stata('data/auto.dta')
print("Data Loaded!")

File: r/1_read.R
#########################################################################
#
# Filename: read.m
#
#
# Purpose: This script reads auto.csv
#
# Created by: adw54
# Created on: 7 Oct 2020
# Last updated on: 7 Oct 2020
#
#########################################################################

library( haven)

## Read data
library( haven)
df <- read_dta("data/auto.dta")
print("Data Loaded!")
